src/ssa.rs

```diff
@@ -246,7 +246,10 @@ impl SSA {
         let mut line_num = 0;
 
         let mut blocks = vec![vec![]];
-        for line in content.as_ref().lines() {
+
+        let contents: String = content.as_ref().to_string().replace("\u{feff}", "");
```

Comment:
What does this character do / why removing it?

Reply:
When you try to open any subtitle file the first char will be \u{feff} and it breaks parsing, so we need to remove it. I don't why it appears when I do `read_to_string`

Reply:
Apparently `\ufeff` is a [byte order mark](https://en.wikipedia.org/wiki/Byte_order_mark) (BOM). Normally this is only set when the file is UTF-16 or UTF-32 encoded. Rust is completely UTF-8 oriented, so `read_to_string` expects the input file to be in UTF-8 and thus does not strip it away. For me that's something that should be sanitized out by the user and not the library